tests/foreman/ui/test_registration.py

```diff
@@ -405,7 +410,7 @@ def test_global_registration_with_gpg_repo_and_default_package(
 @pytest.mark.tier3
 @pytest.mark.usefixtures('enable_capsule_for_registration')
 def test_global_re_registration_host_with_force_ignore_error_options(
-    session, module_activation_key, default_os, default_smart_proxy, rhel7_contenthost
+    session, module_activation_key, rhel7_contenthost
```

Comment:
Here, RHEL7 is hardcoded. In the test above, RHEL8 is hardcoded. Isn't it better to use general contenthost and parametrize  just some RHEL version, if needed?

Reply:
Yes, I agree. Updated all the tests.